src/routers/transactions/products.ts

```diff
@@ -8,14 +8,96 @@ import { StatusCode } from '@/models/statusCodes';
 
 const router = createRouter();
 
+/**
+ * @api {get} /transactions/products/<type>
+ * @description Returns all product transactions of a specified type
+ *              and all their associated attributes.
+ *              Guaranteed fields are:
+ *                - productTransactionUuid
+ *                - datetime
+ *                - toPersonUuid
+ *                - fromPersonUuid
+ *                - amountOfProduct
+ *                - productUnits
+ *                - costPerUnit
+ *                - currency
+ *              Parameters have no effect on this request.
+ *              Associated attributes can be checked via /products/<type>
```

Comment:
isn't it tedious to have to make 2 requests to be able to use product transaction data? for example, you'd make this kind of request, but in order for a client to present it properly, it will also need to make a request for /products/<type> to know what unit is being used. could we just do the join in the core so it's available in this response?

Reply:
also in the docs, it'd be very helpful if you gave an example of what an associated attribute is (ie. density for milk product transactions)

Reply:
This does do the join as well. You can see in the example it's there. I added the line 

```
 *              An example of an associated attribute would be density as
 *              in the example.
```

To make it clearer.